User/Library/UserMisc.c

```diff
@@ -14,10 +14,7 @@ CpuBreakpoint (
   VOID
   )
 {
-  ASSERT (FALSE);
-
-  while (TRUE) {
-  }
+  abort ();
```

Comment:
What happens if we abort rather than deadloop in code injected into firmware, for instance? Or if this happened in OC in a machine using OpenDuet?

More generally, worrying that this change maybe affects wanted behaviour in situations like the above. (Guess @vit9696 , @mhaeuser will have better idea.)

Reply:
I think User library used only in userspace, it must not be injected into firmware, so I personally doesn't see any problem

Reply:
I am with you, this change is already in the userspace debug library only. Ofc! Fine, forget my comment then! Ty.